samples/openapi3/client/petstore/python/tests/test_pet_api.py

```diff
@@ -24,7 +24,7 @@
 
 import urllib3
 
-HOST = 'http://petstore.swagger.io/v2'
+HOST = 'http://localhost/v2'
```

Comment:
i think we will need to do the same for the python aiohttp samples as well. maybe better to simply update the host table. i will do it with another PR.

Reply:
Actually, the aiohttp samples already use the correct `HOST` value: https://github.com/multani/openapi-generator/blob/29033bf62ac202b00ca70dc7bdcba5c58b6bd7f8/samples/openapi3/client/petstore/python-aiohttp/tests/test_pet_api.py#L27